### DISCUSSION TITLE
detect some user tweeted

### DISCUSSION THREAD

**Discussion by AbhimanyuAryan** [tweepy/tweepy#1415] [Questions] [answered]
Is it possible for tweepy to detect some user's new tweet and automatically comment on their new tweets? If yes please refer me for the same in API

**Comment by Harmon758** [comment, marked as answer]
Yes, you can [stream](https://tweepy.readthedocs.io/en/latest/streaming_how_to.html) the Tweets via [the POST statuses/filter endpoint](https://developer.twitter.com/en/docs/tweets/filter-realtime/api-reference/post-statuses-filter) or poll [the GET statuses/user_timeline endpoint](https://developer.twitter.com/en/docs/tweets/timelines/api-reference/get-statuses-user_timeline) using [`API.user_timeline`](https://tweepy.readthedocs.io/en/latest/api.html#API.user_timeline). If by comment, you mean reply, you can use [the POST statuses/update endpoint](https://developer.twitter.com/en/docs/tweets/post-and-engage/api-reference/post-statuses-update) with [`API.update_status`](https://tweepy.readthedocs.io/en/latest/api.html#API.update_status) and provide the `in_reply_to_status_id` parameter.
